### PR TITLE
updated Readme as project is no longer maintained

### PR DESCRIPTION
**Pull request by sahil-rokt**
### Background ###

- Updated the Readme.md to specify the package is no longer maintained.
- Updated Gradle Version to 7.6

Screen Shot 2023-03-14 at 8 59 18 pm

### Checklist: ###

- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have verified that CI completes successfully.
- [x] All insignificant commits have been squashed.

### CODE REVIEW

**Review by thomson-t — APPROVED** [review]

**Review by danial-rokt — APPROVED** [review]

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Tue Jun 23 11:36:52 AEST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
```
Comment (thomson-t) [inline gradle/wrapper/gradle-wrapper.properties]: I think it is better to update to the `7.6.1` patch [release](https://docs.gradle.org/7.6.1/release-notes.html)

**Review by thomson-t — APPROVED** [review]

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (DJingSeo) [reply gradle/wrapper/gradle-wrapper.properties]: Then, https://services.gradle.org/distributions/gradle-7.6.1-bin.zip?

**Comment by DJingSeo** [comment]
I love that you get things done quickly, @sahil-rokt. Much appreciated.

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (sahil-rokt) [reply gradle/wrapper/gradle-wrapper.properties]: updated

**Pull request merged**